src/extension/utils.js

```diff
@@ -553,8 +553,8 @@ export async function updateAuthHeaderRules() {
             type: 'modifyHeaders',
             requestHeaders: [{
               operation: 'set',
-              header: 'x-auth-token',
-              value: authToken,
+              header: 'authorization',
+              value: `token ${authToken}`,
```

Comment:
Switching to `authorization` header because Safari only allows to modify a fixed list of well-known headers.